src/stores/Entities/hooks.ts

```diff
@@ -41,20 +41,15 @@ export const useEntitiesStore_capabilities_adminable = () => {
     );
 };
 
-export const useEntitiesStore_hasSupportRole = () => {
+export const useEntitiesStore_tenantsWithAdmin = () => {
     return useEntitiesStore(
-        useShallow((state) =>
-            Boolean(state.capabilities.admin[ESTUARY_SUPPORT_ROLE])
-        )
-    );
-};
-
-export const useEntitiesStore_hasDemoTenantAccess = () => {
-    const readable = useEntitiesStore_capabilities_readable();
-    return useEntitiesStore(
-        useShallow(() => {
-            return readable.includes(DEMO_TENANT);
-        })
+        useShallow((state) => [
+            ...new Set(
+                Object.keys(state.capabilities.admin).map((tenant) =>
+                    stripPathing(tenant, true)
+                )
+            ),
+        ])
```

Comment:
This is still somewhat in progress.... but need a list of unique entities that are just the "tenant" portion.